redwood-lazylayout-widget/src/commonMain/kotlin/app/cash/redwood/lazylayout/widget/LazyListUpdateProcessor.kt

```diff
@@ -48,7 +48,7 @@ public abstract class LazyListUpdateProcessor<V : Any, W : Any> {
    * The first placeholder ever returned. We use it to choose measured dimensions for created
    * placeholders if the pool ever runs out.
    */
-  private var firstPlaceholder: Widget<W>? = null
+  private var firstPlaceholder: W? = null
```

Comment:
The easiest way to clear a widget reference is to not have a widget reference to begin with.

This file might require a follow-up as our individual row bindings will have a Widget object. I need to validate that in the app once I’ve got the new `detach()` function in place.